maas/plugins/cinder_service_check.py

```diff
@@ -86,19 +86,26 @@ def check(auth_ref, args):
             if '@' in service['host']:
                 [host, backend] = service['host'].split('@')
                 name = '%s-%s_status' % (service['binary'], backend)
-                metric_bool(name, service_is_up)
+            else:
+                name = '%s_status' % service['binary']
 
-        name = '%s_status' % service['binary']
-        metric_bool(name, all_services_are_up)
+            metric_bool(name, service_is_up)
+
+        metric_bool('cinder-volume_status', all_services_are_up)
```

Comment:
This line assumes all services are cinder-volume services. I have an AIO with the following Cinder services with cinder-scheduler currently down:

```
# cinder service-list
+------------------+-------------------------------------------+------+---------+-------+----------------------------+-----------------+
|      Binary      |                    Host                   | Zone |  Status | State |         Updated_at         | Disabled Reason |
+------------------+-------------------------------------------+------+---------+-------+----------------------------+-----------------+
|  cinder-backup   |                   test1                   | nova | enabled |   up  | 2016-07-05T10:40:17.000000 |        -        |
| cinder-scheduler | test1_cinder_scheduler_container-6b2fcb78 | nova | enabled |  down | 2016-07-05T10:38:59.000000 |        -        |
|  cinder-volume   |                 test1@lvm                 | nova | enabled |   up  | 2016-07-05T10:40:11.000000 |        -        |
+------------------+-------------------------------------------+------+---------+-------+----------------------------+-----------------+

```

For the cinder-scheduler check the current version of the plugin gives the following output showing that the scheduler is down
```
# python /usr/lib/rackspace-monitoring-agent/plugins/cinder_service_check.py test1_cinder_api_container-9b5cadb7 --host test1_cinder_scheduler_container-6b2fcb78
status okay
metric cinder-scheduler_status uint32 0
```

This change is adding a cinder-volume metric to the cinder-scheduler check as well as reporting the incorrect status of the cinder-volume service.
```
# python /usr/lib/rackspace-monitoring-agent/plugins/cinder_service_check.py test1_cinder_api_container-9b5cadb7 --host test1_cinder_scheduler_container-6b2fcb78
status okay
metric cinder-scheduler_status uint32 0
metric cinder-volume_status uint32 0
```